cli/commands/benchmark-web-vitals.mjs

```diff
@@ -190,7 +249,7 @@ async function benchmarkURL( browser, params ) {
 				 *
 				 * Click off screen to prevent clicking a link by accident and navigating away.
 				 */
-				await page.click( 'body', { offset: { x: -500, y: -500 } } );
+				await page.click( 'body', { offset: { x: -500, y: -500 } } ); // TODO: Options seems invalid. Should be {delay?: number, button?: MouseButton, clickCount?: number}.
```

Comment:
How so?

[`page.click()`](https://pptr.dev/api/puppeteer.page.click/#parameters) takes [`ClickOptions`](https://pptr.dev/api/puppeteer.clickoptions), which right now only supports `offset`.

Reply:
Humm. Strange. My IDE was complaining here:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/bc3325e1-262a-469f-a8f4-1963d53bbc06)

But only when this branch is checked out. Seems it is caused by importing `Browser` from `puppeteer`. Looking up the definition of `click` it sends me to `lib/types.d.ts`:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/43758031-d11a-49d2-b14d-53706234a19c)

This is what is shown in Puppeteer v19: https://github.com/puppeteer/puppeteer/blob/puppeteer-core-v19.7.1/packages/puppeteer-core/src/api/Page.ts#L2217-L2227

And Puppeteer v19.7.1 is the version showing up in `package-lock.json`.

It seems `ClickOptions` with `offset` was introduced in Puppeteer v20 which we're not currently using.

Reply:
@felixarntz should we also upgrade to Puppeteer v20, maybe in another PR?

Reply:
Not sure what other implications that upgrade would have, but in general upgrading sounds good of course, so +1 to exploring that in another PR.

Reply:
I do see `offset` included in the docs for `ClickOptions` in the docs for v19.7.1: https://github.com/puppeteer/puppeteer/blob/puppeteer-v19.7.1/docs/api/puppeteer.clickoptions.md

Reply:
The TS types don't reflect that accurately (page.click` is not referencing `ClickOptions`, which is why the IDE is confused. Anyway, #61 fixes this automatically.